src/docsfy/api/projects.py

```
@@ -983,12 +997,22 @@ async def _replace_base_variant() -> None:
         )
 
     if plan is None:
         raise RuntimeError("Plan was not generated")
     plan["repo_url"] = source_url
 
+    # Extract auto-detected or provided repo_type from plan
+    detected_repo_type = plan.get("repo_type", "app")
+    if detected_repo_type not in REPO_TYPES:
+        logger.warning(
+            f"[{project_name}] Planner returned unknown repo_type "
+            f"'{detected_repo_type}', defaulting to 'app'"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)